views/block.tmpl

```diff
@@ -55,6 +55,19 @@
 				<div class="fs13 text-secondary pb-1">Block Hash</div>
 				<div class="d-inline-block fs14 break-word rounded font-weight-bold">{{.Hash}}</div>
 			</div>
+			{{if gt (len $.AltBlocks) 0}}
+			<div class="text-left lh1rem py-1">
+				<div class="fs13 text-secondary pb-1">
+					Alternative Block{{if gt (len $.AltBlocks) 1}}s{{end}}
+				</div>
+				{{range $.AltBlocks}}
```

Comment:
This could make the view grow in height quite a bit since `AltBlocks` is unbounded when there are a lot of valid side chains. Perhaps the range should be populating a drop down or a collapsible view which by default shows one entry?